pallets/runtime/testnet/src/runtime.rs

```diff
@@ -100,15 +100,15 @@ include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("polymesh"),
     impl_name: create_runtime_str!("polymath-polymesh"),
-    authoring_version: 1,
+    authoring_version: 0,
     // Per convention: if the runtime behavior changes, increment spec_version
     // and set impl_version to 0. If only runtime
     // implementation changes and behavior does not, then leave spec_version as
     // is and increment impl_version.
-    spec_version: 2000,
+    spec_version: 0,
```

Comment:
is there a reason to reset this ?

Reply:
Yes, to force usage of the `wasm` runtime instead of native runtime on live networks like Alcyone that have spec version > 0.

Reply:
Then we can leave as it is. As Alcyone is on 2004 so existing `spec_version` will work. Anyway, it is fine as well.